### PR TITLE
[DF-6] Login Page

### PR DESCRIPTION
**Pull request by octobrekids**

### CODE REVIEW

File: styles/Global.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+export type PComponentProps = {
+  align?: Align['type'];
+} & React.DetailedHTMLProps<
+  React.HTMLAttributes<HTMLParagraphElement>,
+  HTMLParagraphElement
+>;
+
+export const handleAlignType = (align?: PComponentProps['align']) => {
```
Comment (yee2542) [inline styles/Global.styled.tsx:29]: Missing return type

File: pages/login.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  wrapperCol: { span: 24 },
+};
+
+export type FormLogin = { email: string; password: string };
+export type FormLoginError = { values: FormLogin; errorFields: [] };
+
+interface Props {
+  onSubmit?: (form: FormLogin) => void;
+  onError?: (form: FormLoginError) => void;
+}
```
Comment (yee2542) [inline pages/login.tsx:21]: where this `props` gonna use ?
LoginContainer or LoginWrapper ? maybe it not necessary.

Comment (yee2542) [reply pages/login.tsx:21]: Ahhh I see it for testing, just keep it

**Review by yee2542 — CHANGES_REQUESTED** [review]

File: pages/login.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+const Login: React.FC<Props> = (props) => {
+  const [form] = useForm<FormLogin>();
+
+  const onFinish = (values) => {
+    props.onSubmit && props.onSubmit(values);
+    console.log('Success:', values);
+  };
+
+  const onError = (form) => {
```
Comment (yee2542) [inline pages/login.tsx:31]: risk code, use the shadow name will get mess up

Comment (octobrekids) [reply pages/login.tsx:31]: how should I name it, do you have any examples?

**Review by yee2542 — APPROVED** [review]
Gogo

**Review by yee2542 — APPROVED** [review]
GoGo

**Review by octobrekids — COMMENTED** [review]
Seen

**Pull request merged**